protocols/relay/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 0.21.1
 
+- Automatically configure HOP protocol advertisement based on external addresses, with the ability to override this
+  functionality using `Behaviour::set_status` to explicitly set `Status::{Enable,Disable}` to enable or disable
+  protocol advertisement.
+  See [PR 6154](https://github.com/libp2p/rust-libp2p/pull/6154).
```

Comment:
I think this is a breaking change. Previously, the HOP advertisement was always enabled, whereas now it depends on external addresses being present. The behavior in the end is probably the same (reservations only work if external addresses are present), but the error on the client side will be a different one.